content/W2015/W15-event-websitegitworkshop.md

```diff
@@ -0,0 +1,18 @@
+Title: Github/git and Website Workshop
+Date: 2015-03-1 12:00-17:00
+Category: Events
+Tags: workshop
+Slug: git-website-workshop
+Author: Evy Kassirer
+Summary: Come learn about web design and build your own website, hosted on Github.
+
+Interested in creating your own personal website? Women in Computer Science will teach you the nuts and bolts of web design and how to use git through GitHub! Join us for an afternoon of learning, coding, and snacking! Participants are expected to have basic knowledge of HTML and CSS.
+
+RSVP: There are limited spots available! Please fill out the following form by February 28th. We will let you know if you've been selected by March 3rd. http://goo.gl/forms/ybYZ4uL8Ld
+
+## Workshop Details ##
+
++ **Who:** Cis and trans women and non-binary folks
++ **What:** Github/git and website workshop
++ **Where:** Details will be emailed to participants
```

Comment:
will that confuse participants who are trying to find where it is later, but forget that they have an email or something? That's why I decided not to put TBA, but I agree that TBA is cleaner